packages/block-library/src/navigation-submenu/edit.js

```diff
@@ -529,6 +550,11 @@ export default function NavigationSubmenuEdit( {
 
 	const ParentElement = openSubmenusOnClick ? 'button' : 'a';
 
+	function transformToLink() {
```

Comment:
useCallback could help avoid some re-rendering

Reply:
Did you notice a performance problem here? If not I generally [adhere to KCD](https://kentcdodds.com/blog/usememo-and-usecallback#why-is-usecallback-worse).